### ISSUE TITLE
When trying to create a village for the 2018 camp it shows up under 2017 instead

### ISSUE THREAD

**Issue by netravnen** [bornhack/bornhack-website#156]
See the title

_NB duplicates has been deleteded already._

**Comment by netravnen** [comment]
Resulted in urls
1. https://bornhack.dk/bornhack-2017/villages/prosa-village-1/ and
2. https://bornhack.dk/bornhack-2017/villages/prosa-village-2/
When 
1. Going to https://bornhack.dk/bornhack-2018/villages/
2. Clicking "create a new bornhack 2018 village "

**Comment by valberg** [comment]
@netravnen Is this still an issue? Looks like the villages are in the correct places now?

**Comment by netravnen** [comment]
@valberg

Apparently not... (?)

Just tried again. Works... (now I'm not sure if i failed to verify what I did the 1st and 2nd time back before I created this ticket... )

I saw the two following entries where created:
1. https://bornhack.dk/bornhack-2018/villages/prosa-village-3/
2. https://bornhack.dk/bornhack-2019/villages/prosa-village-4/

... so _must_ have done something wrong as no git commits is listed at https://github.com/bornhack/bornhack-website/commits/master ... ... ... (now I'm just utterly confused)

But yes. The issue is non-outstanding. So ticket can be closed. :)

.netravnen
